cmake/Modules/GrPython.cmake

```diff
@@ -165,7 +165,7 @@ if not install_dir:
     else:
         scheme = 'posix_prefix'
     install_dir = sysconfig.get_path('platlib', scheme)
-    prefix = sysconfig.get_config_var('prefix')
+    prefix = sysconfig.get_path('data')
```

Comment:
probably the righter thing! And since it doesn't break on the debianoids, this seems fine. Let's wait for the conda builds (but these we could fix ourselves, no losses), but this: really LGTM.